src/frontend/Player/createPlyrPlayer.ts

```diff
@@ -32,10 +32,14 @@ export const createPlyrPlayer = async (
     );
   } catch (e) {}
 
-  const { intl } = new IntlProvider({
-    locale: localeCode,
-    messages: translatedMessages,
-  }).getChildContext();
+  const cache = createIntlCache();
+  const intl = createIntl(
```

Comment:
I think you're supposed to do this in `index` so you can reuse the same `intl` object throughout the app

Reply:
which `index` ?

Reply:
In `index.tsx` where we have the `IntlProvider`

Reply:
And what about using `useIntl` in `<VideoPlayer />` and then inject in `createPlayer` method the intl shape ?

Reply:
I made a modification in this commit https://github.com/openfun/marsha/pull/479/commits/6ff057bc376674b296381a7f3ceddc257d4c8b00 if it's ok for you I will squash it.

Reply:
IMO it's cleaner to use the `createIntl` technique to avoid passing a meaningless argument to `createPlyrPlayer` and `createPlayer` (who already has a kinda big signature)

But if you prefer it this way it's OK for me too

Reply:
Like this ? https://github.com/openfun/marsha/pull/479/commits/68a705c7bb78cd334b3b28e437bd77cdccbe0b9b